cluster-autoscaler/FAQ.md

```diff
@@ -30,10 +30,22 @@ HPA-created pods have a place to run.
 If the load decreases HPA will stop some of the replicas. As the result some nodes may start to be 
 underutilized or completely empty and then CA will delete such unneded nodes.
 
+### What are the key best practices for running Cluster Autoscaler?
+
+* Do not modify the nodes. All nodes within the same node group should have the same capacity, labels and system pods dunning on them
+* Specify requests for your pods.
+* Use Pod Disruption Budgets to prevent pods from being deleted (if needed).
```

Comment:
Inconsistent spelling of pdb - in a later question it's "PodDisruptionBudget" (no spaces).